### PR TITLE
Dropbox adapter

### PR DESCRIPTION
**Pull request by Praxie**
Dropbox adapter for the CollectionFS package.

### CODE REVIEW

File: packages/dropbox/package.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Package.describe({
+  name: 'cfs:dropbox',
+  version: '0.0.1',
+  summary: 'Dropbox storage adapter for CollectionFS',
+  git: 'https://github.com/CollectionFS/Meteor-cfs-dropbox.git',
```
Comment (rhyslbw) [inline packages/dropbox/package.js]: @aldeed this should just be the main repo now right?

**Comment by rhyslbw** [comment]
Thanks @Praxie . Just a minor spelling mistake with the naming of LICENCE.MD and the repo link in package.js

File: packages/dropbox/package.js (hunk 1 above)
Comment (raix) [inline packages/dropbox/package.js]: can we do something like : `https://github.com/CollectionFS/Meteor-CollectionFS.git/packages/dropbox`? will atmosphere support this pattern @tmeasday?
In short we have all core packages in one repo - in an attempt to make the project easier to contribute to and have an overview over. All the packages are placed in a Meteor QA app for development, making it easier to make contributions.

**Comment by raix** [comment]
Awesome work you guys!!!!! - you should write about it on a blog or forums.meteor.com

File: packages/dropbox/package.js (hunk 1 above)
Comment (tmeasday) [inline packages/dropbox/package.js]: @raix I think just leave as the .git URL and use the `readme` field to set a specific read me. Does that work?

**Comment by Praxie** [comment]
@rhyslbw thanks. So the right repo link is 'https://github.com/CollectionFS/Meteor-CollectionFS/tree/master/packages/dropbox', isn't it?

**Comment by aldeed** [comment]
:+1:

**Comment by rhyslbw** [comment]
@Praxie Yep that link should work. Just need to fix the spelling of the licence file. Should be 'LICENSE'

**Pull request merged**